README.md

```
@@ -0,0 +1,24 @@
+# DevBlog
+
+Minimalist Jekyll blog with automated GitHub Actions deployment.
+
+**Live site:** https://texarkanine.github.io/devblog/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)